content/en-bulletin.md

```diff
@@ -40,11 +40,11 @@ _Spanish teacher, Antonia Asencio (seating, center), and students, proudly showi
 
 ## MACILE Spanish Curriculum
 
-An innovative active approach to deepen understanding of the language through the history and folklore of the Dominican Republic. Dr. Rosario Montelongo de Swanson wrote a book exclusively for MACILE.
+The Spanish curriculum, featured in 2008, is an innovative active approach to deepen understanding of the language through the history, literature, and folklore of the Dominican Republic and Latin America. It was written exclusevely for MACILE by Dr. Rosario Montelongo de Swanson.
```

Comment:
I like these changes. I suggest rewriting the first sentence as, 

_Debuted in 2008, MACILE's Spanish curriculum is an ....._

spelling correction: Exclusively